README.md

```diff
@@ -43,6 +43,8 @@ roles responsible for building and running apps and operating infrastructure.
 
 For more details and user stories, see [introduction.md](./introduction.md).
 
+[Relations between various OAM resources](./assets/relations_between_various_OAM_resources.png) 
```

Comment:
We don't need this README now, just keep it in 2.overview_and_terminology.md

Reply:
Ok